.prettierrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  bracketSpacing: false,
+  jsxBracketSameLine: true,
+  printWidth: 120,
+  semi: true,
+  singleQuote: true,
+  tabWidth: 2,
+  trailingComma: "none",
```

Comment:
Ten przecinek na koncu to mala dogodnosc, bo mozesz odrazu dodac nastepny element do arraya / obiektu, zamiast wrzucac samemu przecinek. W zaden sposob nie wplywa na dzialanie kodu, wiec nie wiem po co go wyrzucac. Uwazasz, ze lepiej wyglada bez tego?

Reply:
Hm... ja jestem przeciwniczką pisania kodu "na zapas", bo może ktoś kiedyś tego użyje. Jeśli ktoś ma potrzebę dodania kolejnego elementu do tablicy/obiektu, to go wtedy dopisze. Prawdopodobnie stąd mój brak zrozumienia dla tej reguły przecinka na końcu ;) Raczej powinno się odchodzić od takich niepotrzebnych dodatków aniżeli je namnażać. Ale jeżeli będziecie chcieli ten przecinek, to nie będę się upierała przy swoim ;)